.github/workflows/CI.yml

```
@@ -515,6 +518,8 @@ jobs:
             public.ecr.aws/${{ env.ECR_REPO }}:latest
           cache-from: type=registry
           cache-to: type=inline
+          platforms : linux/amd64, linux/arm64
+          build-args: ${{ needs.e2etest-preparation.outputs.version }}
```

```suggestion
          build-args: VERSION=${{ needs.e2etest-preparation.outputs.version }}
```
Doesn't this need to know what argument to assign itself to?